geographies/views.py

```diff
@@ -175,8 +177,9 @@ def post(self, request, *args, **kwargs):
         elif end_limit: # case3: only end_limit is present
             queryset = queryset[:int(end_limit)]
 
-        count = self.request.POST.get("count", "False")
-        if count in ["True","true","t","T","Yes","yes","Y","y"]:
+        count = self.request.POST.get("count", "False") # POST param 'count', default value is string "False"
```

Comment:
change the variable name as count suggests something of integer type in nature but here it's related to string.

Reply:
Code outdated. count is no longer in use